packages/koa-prometheus/README.md

```diff
@@ -145,6 +145,26 @@ This library also utilises `prom-client`'s `collectDefaultMetrics` &
 [`node-prometheus-gc-stats`](https://github.com/SimenB/node-prometheus-gc-stats)
 to collect CPU & Garbage Collection stats
 
+#### Service standard metrics
+
+To comply with Uswitch [service standard]() monitoring, enable service standards:
```

Comment:
Bear in mind this is an open source repo so `Uswitch` service standards may not mean anything

Reply:
Hopefully we open source those too